core/commands/keystore.go

```diff
@@ -556,3 +543,20 @@ func keyOutputListEncoders() cmds.EncoderFunc {
 		return nil
 	})
 }
+
+func daemonNotRunning(req *cmds.Request, env cmds.Environment) error {
```

Comment:
note: this function is also copy-pasted in `init.go`, we may want to deduplicate.

Reply:
Deduplicated by exporting this.  There may be a better file to define this function in, but left it here for now since this is the only place in this package that uses it.